DependencyInjection/Configuration.php

```diff
@@ -963,4 +1062,119 @@ public function getConfigTreeBuilder()
 
         return $treeBuilder;
     }
+
+    private function handlerTypeAcceptedOptionsValidation(array $v)
+    {
+        if (!array_key_exists('type', $v)) {
+            return $v;
+        }
+
+        if (!array_key_exists(strtolower($v['type']), $this->acceptedParamsByHandlerType)) {
+            return $v;
+        }
+
+        // @todo array_keys should be converted to lowercase?
```

Comment:
I don't think so? I don't think there is an expectation that this stuff is case insensitive generally speaking, but I might be mistaken.